doc/man3/EVP_sm3.pod

```diff
@@ -42,8 +42,8 @@ L<EVP_DigestInit(3)>
 
 =head1 COPYRIGHT
 
-Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
-Copyright 2017 Ribose Inc. All Rights Reserved.
+Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2017-2018 Ribose Inc. All Rights Reserved.
```

Comment:
Drop the change in copyright. The Ribose claim only relates to the original submission, so doesn't get extended by later modifications. The OpenSSL claim will get updated automatically as part of the release process (we have scripts to do this).

Reply:
Okay, understood. Will fix that later